HHS/Head-Start-TTADP
# Test HTML with CSS

## Description of change
One of my little projects for this year is to add some additional automated accessibility testing. Inspired by the blog post linked in the file, I've added the basics of some "testing accessibility in HTML via CSS"

This is a small feature that allows developers to test accessibility by setting a local environment variable which will produce ugly red borders around inaccessible elements.

We can expand it further using the parent project, [revenge.css ](https://github.com/Heydon/REVENGE.CSS/blob/master/revenge.css). 

## How to test

In frontend/.env set 
REACT_APP_INCLUDE_ACCESSIBILITY_CSS=false
to 
REACT_APP_INCLUDE_ACCESSIBILITY_CSS=true

and restart your frontend app.

You will see the red error boxes around the links-as-buttons on the /landing page AR table

## Issue(s)


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
